scripts/generate_config_tests.py

```diff
@@ -80,7 +80,7 @@ def dependencies_of_setting(cfg: config_common.Config,
     be negated by prefixing them with '!'. This is the same syntax as a
     depends_on directive in test data.
     """
-    #pylint: disable=too-many-return-statements
+    #pylint: disable=too-many-branches,too-many-return-statements
```

Comment:
Nitpick: instead of silencing the warning, we could move the TLS setting processing to a separate function.